samples/unpeelingOnion/src/core/validation.ts

```diff
@@ -0,0 +1,81 @@
+//////////////////////////////////////
+/// Validation
+//////////////////////////////////////
+
+export const enum ValidationErrors {
```

Comment:
Validation is an outdated approach. Should focus on pare -> https://github.com/colinhacks/zod

Reply:
Yes, I’m planning to use it in my other samples, here I focused more on vanilla approach.